HLTrigger/Phase2/python/HLT_75e33/psets/pSetPvClusterComparerForIT_cfi.py

```diff
@@ -3,6 +3,6 @@
 pSetPvClusterComparerForIT = cms.PSet(
     track_chi2_max = cms.double(20.0),
     track_prob_min = cms.double(-1.0),
-    track_pt_max = cms.double(30.0),
+    track_pt_max = cms.double(100.0),
```

Comment:
This is an actual change, and is used (only) by
```python
pixelVertices = cms.EDProducer("PixelVertexProducer",
    Finder = cms.string('DivisiveVertexFinder'),
    Method2 = cms.bool(True),
    NTrkMin = cms.int32(2),
    PVcomparer = cms.PSet(
        refToPSet_ = cms.string('pSetPvClusterComparerForIT')
    ),
    PtMin = cms.double(1.0),
    TrackCollection = cms.InputTag("pixelTracks"),
    UseError = cms.bool(True),
    Verbosity = cms.int32(0),
    WtAverage = cms.bool(True),
    ZOffset = cms.double(5.0),
    ZSeparation = cms.double(0.005),
    beamSpot = cms.InputTag("offlineBeamSpot")
)
```

Given that these are pixel tracks (the the pR resolution is probably not great up to 100 GeV), and that the configuration we have always used for PF (so, Jets/MET, b-tag, and possibly taus) has 30 GeV, I would suggest to keep the current value.

Reply:
From my experience rising it to pT>50 GeV improves the tagging power, increasing the fraction of events for which the leading reco is matched to the true PV. Also it has no effect on the vertex finding itself, only on the final sorting.

Reply:
So this is the recommended setting that everybody should have been using (at least its what Adriano put in his recipe). However I understand that the impact is much more strong on Jets than E/gamma so we can revert. Adriano, how big a change is this? Is it something small enough that your tracking TDR section still holds with this reverted, I mean theres not too much we can do if not but still it would be nice to get a statement on the expected impact.

Reply:
Yes track_pt_max=100GeV is the recommended setting since a while, used for tracking and for sure for Jets and b-tagging. The effect is mild anyway since it is not affecting the tagging power of the trimmed vertices as a whole (it only moves the position of the true vertex from 0 to 1 or 2 for some events). I can run some tests to check it more quantitatively.